github-runner-manager/src/github_runner_manager/openstack_cloud/openstack_cloud.py

```diff
@@ -148,19 +148,28 @@ class OpenstackCloud:
     instance_id. It is the same as the server name.
     """
 
-    def __init__(self, credentials: OpenStackCredentials, prefix: str, system_user: str):
+    def __init__(
+        self,
+        credentials: OpenStackCredentials,
+        prefix: str,
+        system_user: str,
+        proxy_command: str | None = None,
+    ):
         """Create the object.
 
         Args:
             credentials: The OpenStack authorization information.
             prefix: Prefix attached to names of resource managed by this instance. Used for
                 identifying which resource belongs to this instance.
             system_user: The system user to own the key files.
+            proxy_command: gateway argument for fabric Connection. Similar to ProxyCommand in
```

Comment:
The `gateway` should be in caps? Since it is the start of a sentence.
Maybe `The gateway argument ...`.